src/test/java/com/google/javascript/clutz/DeclarationSyntaxTest.java

```diff
@@ -66,6 +67,8 @@ public boolean accept(File dir, String name) {
           "--noImplicitAny",
           "--strictNullChecks");
 
+  private static final int MAXIMUM_TIME_TO_RUN_TSC_IN_SECONDS = 2;
```

Comment:
I'd inline this, it's not like you'll reuse the constant, and it's perfectly readable with the TimeUnit.SECONDS below.